### DISCUSSION TITLE
so-status errors for The Hive and Cortex

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#7994] [Unsupported Versions] [answered]
Hello!

I run SO 2.3.120 and my understanding was, that The Hive and Cortex have been removed completely. However, after a reboot of my master node I get errors in so-status and the SOC grid overview:

![image](https://user-images.githubusercontent.com/81571847/169767848-a0a802f7-2aff-4cbd-a54c-6403c186357e.png)

How can I get rid of these errors?

Cheers, Ben

**Comment by dougburks** [comment, marked as answer]
soup should have removed those entries from so-status.conf using the following command:
```
sed -i '/so-thehive-es/d;/so-thehive/d;/so-cortex/d' /opt/so/conf/so-status/so-status.conf
```

You can run that command manually and you might want to check your soup log to see if there are any other errors.

**Reply by ben-sec** [reply]
The above command helped to remove The Hive and Cortex. However, I found the following error in root/soup.log for the last soup run:

ERROR: salt-master could not be reached.

**Reply by dougburks** [reply]
> ERROR: salt-master could not be reached.

That's most likely just a timing issue. If services appear to be running properly now, then it's probably nothing to be concerned about.
